### ISSUE TITLE
Allow /lobby for server owners

### ISSUE THREAD

**Issue by Reimagining** [Minehut/Meta#662]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

Both (Java and Bedrock)

**Describe the feature**

Give server owners and players the freedom to use /lobby without being sent to the minehut lobby.

**How the feature is useful**

/lobby is a command used by various big and small servers to go back to the main world/area of a server/network. Server owners sometimes suffer the consequence of new minehut players and/or players that are not used to different commands that do /lobby and get confused of what happened when they are sent back to the minehut hub and sometimes even lose the server they were on. 

It would be very beneficial to allow minehut server owners to use /lobby and possibly even /hub for their own usage; as they are common aliases for server "Spawns". Especially when a new "minehutter" or external player gets on, it can get confusing. I understand the need of this command, so I propose keeping /hub or even better, try and use another one such as /mhhub. But at least /lobby should be changed

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by DebitCardz** [comment]
So how would people go back to the main lobby?

**Comment by OGContent** [comment]
> So how would people go back to the main lobby?
> Making a new proxy command (/mhhub) is just very unintuitive for new players to understand, I feel like you're making multiple gamemodes on your server so you want people to use /lobby to get back to the main lobby of your server but this really wouldn't benefit the majority of Minehut players and would instead make their experience worse.
> 
> If you can give reasons as to why this would benefit **everyone** on Minehut I will reconsider.

/hub
/minehut:lobby
/minehut:hub

**Comment by DebitCardz** [comment]
Why would this be useful for the general playerbase though?

**Comment by OGContent** [comment]
> Why would this be useful for the general playerbase though?

Well most of the servers would like a command like /hub for their own instead of /spawn, and I was just giving answers so server owners would like it

**Comment by DebitCardz** [comment]
I personally find it frustrating when servers don't support both /lobby & /hub and I have to find which one works.

**Comment by Reimagining** [comment]
Most of player base's are loyal to one server for a given time. I understand the concept of connecting the network of servers that minehut has, but normally when you host a server, you would like to have an isolated and controllable environment. Having /lobby is just one of many commands that is very important to a server, and having it reserved for minehut/their service/promotion is a bit annoying and can affect the player base. So I do believe for the majority of people would find this feature more than beneficial, it's not as if the command is getting removed, but the majority of players aren't on minehut to go back to forth to the lobby as they are to stay on one specific server. It's a simple command that wouldn't be so hard to provide, as much as it affects server control imo.

**Comment by LiftenMH** [comment]
Maybe a setting in the minehut panel like "Disable Community" which could disabled /hub, /lobby, other minehut commands, finding players on the server, and joining from the lobby.

**Comment by Reimagining** [comment]
Yeah, I'm not sure how, but owners should be able to own an independent server rather than just a minehut server that is linked and advertised to them in every way.

**Comment by CerialPvP** [comment]
its a good thing if you have a private server and you dont want randoms joining, but you know, /whitelist exists there for a reason

**Comment by Reimagining** [comment]
I would like to offer a rebuttal.

Allowing /lobby or /hub on servers or subnetworks on minehut, but then send a chat to the target player with text such as "Want to go to the minehut lobby? Please use /mhub" or something of the sort. With this option, a crucial command and server element, is free to use amongst the server owners and minehut clientele while showing minehut players how to join between the two lobbies.
